docs/professions/overview.md

```diff
@@ -10,7 +10,6 @@ An aisling may only learn 1 primary profession, but all 3 secondary professions
 - [Armorsmithing](./armorsmithing.md)
 - [Enchanting](./enchanting.md)
 - [Jewelcrafting](./jewelcrafting.md)
-- [Tailoring](./tailoring.md)
```

Comment:
Can you update the profession description at the top of the page to say there are only 5 primary professions?